lib/utils/misc.ts

```diff
@@ -4,17 +4,24 @@ import { SupportedCurrencySymbol } from '/lib/types.ts';
 let BASE_URL = typeof window !== 'undefined' && window.location
   ? `${window.location.protocol}//${window.location.host}`
   : '';
+let CUSTOM_TITLE = '';
+let CUSTOM_DESCRIPTION = '';
+let HELP_EMAIL = '';
 
 if (typeof Deno !== 'undefined') {
   await import('std/dotenv/load.ts');
 
   BASE_URL = Deno.env.get('BASE_URL') || '';
+
+  CUSTOM_TITLE = Deno.env.get('CUSTOM_TITLE') || '';
+  CUSTOM_DESCRIPTION = Deno.env.get('CUSTOM_DESCRIPTION') || '';
+  HELP_EMAIL = Deno.env.get('HELP_EMAIL') || '';
```

Comment:
Can you please define these as `const` and remove the initial `let` definitions above?

Reply:
I could not use consts there, assigments are scoped in if statement checking for Deno. I could only clean it up. May I missing something?

Reply:
Ah, dang it. You're right, I didn't realize this was inside the scoped `if`. In that case, I'd prefer it as it was before, then.